spec/factories/players.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :player do
-    match ""
-house "MyString"
+    association :match
```

Comment:
Yes, if you do not pass a previously created one, it creates, no matter if using `create` or `build`.